go/vt/vttablet/onlineddl/executor.go

```diff
@@ -3098,6 +3098,25 @@ func shouldCutOverAccordingToBackoff(
 	return false, false
 }
 
+// getInOrderCompletionPendingCount returns a count of migrations that must cut-over in-order, before the
+// provided migration is able to proceed. This count is relevant only if the migration uses the
+// --in-order-completion option.
+func getInOrderCompletionPendingCount(onlineDDL *schema.OnlineDDL, pendingMigrationsUUIDs []string) uint64 {
+	if len(pendingMigrationsUUIDs) == 0 {
+		return 0
+	}
+	var pendingCount uint64
+	for _, pendingMigrationsUUID := range pendingMigrationsUUIDs {
+		if pendingMigrationsUUID == onlineDDL.UUID {
+			// found all migrations we must wait for if
+			// we found ourself in the pending list.
+			break
+		}
+		pendingCount++
+	}
+	return pendingCount
```

Comment:
What does it mean if our UUID is not in the list? Is that not possible? If it is, does that mean that we're not waiting on anything in the current list? I'm not sure how it's managed in total, so may be a non-issue.